schema/cpacs_schema.xsd

```diff
@@ -21533,6 +21565,12 @@ jonas.jepsen@dlr.de
 		<xsd:complexContent>
 			<xsd:extension base="complexBaseType">
 				<xsd:sequence>
+					<xsd:element minOccurs="0" name="elastic" type="aeroPerformanceMapElasticType">
```

Comment:
This node ended up in the wrong location. Should be put in aeroPerformanceMap as optional node.